packages/server/src/utils/logger.js

```diff
@@ -0,0 +1,43 @@
+import pino from 'pino'
+import { merge } from '@ditojs/utils/lib/object/merge'
```

Comment:
@ptoussai didn't notice, but this should've been just `import { merge } from '@ditojs/utils'